docs/content/stable/reference/configuration/yb-tserver.md

```diff
@@ -1856,6 +1856,18 @@ Default: 1024
 
 Default: true
 
+##### yb_insert_on_conflict_read_batch_size
+
+{{<tags/feature/ea idea="1455">}} Set the level of batching for [INSERT ... ON CONFLICT](../../../api/ysql/the-sql-language/statements/dml_insert/#on-conflict-clause). Set to 0 to disable batching. Batching is always disabled for the following:
```

Comment:
Link this from the [dml_insert](https://deploy-preview-26003--infallible-bardeen-164bc9.netlify.app/stable/api/ysql/the-sql-language/statements/dml_insert/#column-values-1) page as well.